src/Adapter/ParameterContainer.php

```diff
@@ -64,37 +64,37 @@ class ParameterContainer implements Iterator, ArrayAccess, Countable
      */
     public function __construct(array $data = [])
     {
-        if ($data) {
+        if ($data !== []) {
             $this->setFromArray($data);
         }
     }
 
     /**
      * Offset exists
      *
-     * @param  string $name
+     * @param  string $offset
      * @return bool
      */
-    #[ReturnTypeWillChange]
-    public function offsetExists($name)
+    #[Override] #[ReturnTypeWillChange]
+    public function offsetExists($offset)
     {
-        return isset($this->data[$name]);
+        return isset($this->data[$offset]);
     }
 
     /**
      * Offset get
      *
-     * @param  string $name
+     * @param  string $offset
      * @return mixed
      */
-    #[ReturnTypeWillChange]
-    public function offsetGet($name)
+    #[Override] #[ReturnTypeWillChange]
```

Comment:
These need to be one per line. If I had to guess this was a Rector change? They still need to be one per line.